apps/ui/BaseWindow.cpp

```diff
@@ -259,9 +258,7 @@ void BaseWindow::display()
     const auto& camera = _brayns.getEngine().getCamera();
     renderInput.windowSize = windowSize;
     renderInput.position = camera.getPosition();
-    renderInput.target = camera.getTarget();
-    renderInput.up = camera.getUp();
-
+    renderInput.quat = camera.getQuaternion();
```

Comment:
```suggestion
    renderInput.quaternion = camera.getQuaternion();
```

Reply:
As mentioned in other comment, this should be called orientation, not quaternion

Reply:
Done